onnx/__init__.py

```diff
@@ -29,3 +29,16 @@ def load(obj):
         with open(obj, 'rb') as f:
             model.ParseFromString(f.read())
     return model
+
+
+def load_from_string(obj):
```

Comment:
Maybe not to add a separated function but use `os.path.exists` in `load` to distinguish a path vs. a string?

Reply:
I think it's safer to separate it from load. :-)

Reply:
I was ok with adding a separate function yesterday :-) But today there is this PR #265 adding an additional check in the `load` function, and it makes me prefer to merge this into `load`.

Reply:
Sure, with this check, it is safer to do so.

Reply:
Actually it's better to have it as a separate function imho. It's weird that load('typo in my file name') will try to parse the string as protobuf :)